doc/development.rst

```diff
@@ -24,3 +24,57 @@ Python compatibility
 --------------------
 
 We follow `numpy's Python support policy <https://numpy.org/neps/nep-0029-deprecation_policy.html>`_.
+
+Release process
+---------------
+
+1. Update the version number in ``petab/version.py``.
+
+2. Update the changelog in ``doc/CHANGELOG.md``.
+
+3. Create a pull request with the changes to the main branch.
```

Comment:
Is this from `develop`? Or some `release_v101` like
```
git checkout develop
git pull
git checkout -b release_v101
git add petab/version.py doc/CHANGELOG.md
git commit -m 'bump version; update changelog'
```

If the latter then some more info is needed I think. "simple merge" to main, then merge main into develop.

Reply:
I'd leave that to the releasing person and assume some basic familiarity with git/github.